.travis.yml

```diff
@@ -4,5 +4,18 @@ jdk:
 before_install:
   - sudo apt-get install libaio1 # Needed to run mysqld for integration tests
 script: mvn verify
-# TODO: Need to add a deploy line when we have bintray access
-# See: http://docs.travis-ci.com/user/customizing-the-build/
+# TODO: Add coveralls support
+after_success:
+  - test "${TRAVIS_PULL_REQUEST}" == "false" && test "${TRAVIS_TAG}" != "" && mvn deploy --settings travis/settings.xml
+#  - mvn coveralls:report
```

Comment:
We plan on adding coveralls support eventually. I'll move the todo line from line 7